ref_ts/views.py

```diff
@@ -208,6 +209,7 @@ def create_ref_time_series(request, *args, **kwargs):
         context = {'resource_creation_error': "Error: failed to create resource." }
         return render_to_response('pages/create-ref-time-series.html', context, context_instance=RequestContext(request))
 
+@api_view(['GET'])
```

Comment:
Did you test if the view function works with the added decorator when download is initiated from the resource landing page?